airbyte-ci/connectors/live-tests/pyproject.toml

```diff
@@ -57,8 +57,11 @@ select = ["I", "F"]
 known-first-party = ["connection-retriever"]
 
 [tool.poe.tasks]
-format = "ruff format src"
 test = "pytest tests"
-lint = "ruff check src"
 type_check = "mypy src --disallow-untyped-defs"
-pre-push = ["format", "lint", "test", "type_check"]
+pre-push = []
+
+[tool.airbyte_ci]
+optional_poetry_groups = ["dev"]
+poe_tasks = []
+required_environment_variables = ["DOCKER_HUB_USERNAME", "DOCKER_HUB_PASSWORD"]
```

Comment:
Can you tell me more about how we are using those credentials in live-tests? Are we using any private images today, and/or publishing? Are we logging in to dockerhub in the flow, and do we have to do it? /cc @alafanechere